examples/test-c-client.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * SRT - Secure, Reliable, Transport
+ * Copyright (c) 2017 Haivision Systems Inc.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; If not, see <http://www.gnu.org/licenses/>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+
+#include "srt.h"
+
+int main( int argc, char** argv )
+{
+    int ss, st;
+    struct sockaddr_in sa;
+    int yes = 1;
+    const char message [] = "This message should be sent to the other side";
+
+    printf("srt startup\n");
+    srt_startup();
+
+    printf("srt socket\n");
+    ss = srt_socket(AF_INET, SOCK_DGRAM, 0);
```

Comment:
Please use `srt_create_socket()`. The `srt_socket()` function is deprecated.

Reply:
I'll try!
This it's only a dirty port of the examples in the previous PR.